server/helpers/identification.js

```diff
@@ -0,0 +1,14 @@
+import { User } from '../database/models';
+import responseMessage from './responseMessage';
+
+const getUserId = async (param, response) => {
+  try {
+    const field = (/\D/g.test(param)) ? { email: param } : { id: param };
```

Comment:
what happens when there is a number in the email?

Reply:
it will still see it as an email

Reply:
the only time it regular expression returns false is if the param is a number